src/main/java/htsjdk/tribble/AsciiFeatureCodec.java

```diff
@@ -49,14 +50,19 @@ public boolean isDone(final LineIterator lineIterator) {
     }
 
     @Override
-    public LocationAware makeIndexableSourceFromStream(final InputStream bufferedInputStream) {
-        final PositionalBufferedStream pbs;
-        if (bufferedInputStream instanceof PositionalBufferedStream) {
-            pbs = (PositionalBufferedStream) bufferedInputStream;
+    public LocationAware makeIndexableSourceFromStream(final InputStream inputStream) {
```

Comment:
Can you file a ticket against `LocationAware` to either make all implementation obey the interface contract, or change the interface to reflect the actual behavior?